UIKit/Location Controller/TWTLocationController.h

```diff
@@ -0,0 +1,41 @@
+//
+//  TWTLocationController.h
+//  Toast
+//
+//  Created by Steve Foster on 2/3/15.
+//  Copyright (c) 2015 Two Toasters, LLC. All rights reserved.
+//
+
+@import Foundation;
+
+@class CLLocation;
+
+
+typedef NS_ENUM(NSInteger, TWTLocationControllerStatus) {
+    TWTLocationControllerAuthorizationStatusNotDetermined = 0,
+    TWTLocationControllerAuthorizationStatusRestricted,
+    TWTLocationControllerAuthorizationStatusDenied,
+    TWTLocationControllerAuthorizationStatusAuthorizedAlways,
+    TWTLocationControllerAuthorizationStatusAuthorizedWhenInUse,
+    TWTLocationControllerErrorPermissionRequired,
+    TWTLocationControllerErrorPermissionDenied,
+    TWTLocationControllerErrorPermissionRestricted,
+    TWTLocationControllerErrorUpdateFailed,
+    TWTLocationControllerErrorUpdateTimedOut
```

Comment:
Some of these are error codes and some are authorization statuses. I don’t see why anyone would need an authorization status here that’s different than the CoreLocation one. As such, just make these all error codes. I’d also name this enum `TWTLocationControllerErrorCode` and prefix each error code with the same.